### DISCUSSION TITLE
More verbose report

### DISCUSSION THREAD

**Discussion by hreisin-qontigo** [coverlet-coverage/coverlet#1410] [Q&A] [answered]
Hi, apologies if this is an answered question. 

Is it possible to know which test(s) hit a given line not only the number of hits?

Thanks

**Comment by daveMueller** [comment, marked as answer]
Hi @hreisin-qontigo, there currently is no support for per-test coverage.
In the issue [https://github.com/coverlet-coverage/coverlet/issues/973](https://github.com/coverlet-coverage/coverlet/issues/973) you can find some more insights on the topic.
